CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed the `OnAuthorityLossImminent` Actor event.
 
 ### Features:
+- You can now disable logging to spatial from the GUI (Project Settings -> Logging). The command line argument -NoLogToSpatial can still be used for that as well.
```

Comment:
Do we generally add things at the top or the bottom? I think I prefer bottom so it's chronological from the oldest to the newest, but can be convinced either way.